apps/web/next.config.js

```diff
@@ -717,8 +719,12 @@ const nextConfig = {
 if (!!process.env.NEXT_PUBLIC_SENTRY_DSN) {
   plugins.push((nextConfig) =>
     withSentryConfig(nextConfig, {
+      org: process.env.SENTRY_ORG,
+      project: process.env.SENTRY_PROJECT,
+      authToken: process.env.SENTRY_AUTH_TOKEN,
+      // Upload a larger set of source maps for prettier stack traces (increases build time)
+      widenClientFileUpload: true,
       autoInstrumentServerFunctions: false,
-      hideSourceMaps: true,
```

Comment:
Deprecated!

> The `hideSourceMaps` option was removed without replacements. The SDK emits hidden sourcemaps by default.

https://docs.sentry.io/platforms/javascript/guides/nextjs/migration/v8-to-v9/#removed-apis